experimental/packages/opentelemetry-sdk-node/package.json

```diff
@@ -44,22 +44,22 @@
     "access": "public"
   },
   "dependencies": {
-    "@opentelemetry/api-metrics": "0.29.1",
-    "@opentelemetry/core": "1.3.0",
-    "@opentelemetry/instrumentation": "0.29.1",
+    "@opentelemetry/api-metrics": "0.29.2",
+    "@opentelemetry/core": "1.3.1",
+    "@opentelemetry/instrumentation": "0.29.2",
     "@opentelemetry/resource-detector-aws": "~1.0.0",
     "@opentelemetry/resource-detector-gcp": "~0.26.0",
```

Comment:
I guess we should move them also to newer versions - or remove it at all to avoid the chicken/egg issue here.

Reply:
I support removing them from here, add adding all resource detectors to the `@opentelemetry/auto-instrumentations-node` package from contrib, similar to instrumentations.

Reply:
For now I'll update them, but we should talk about removing them. I think that isn't something for this PR though.